### ISSUE TITLE
Push to github.io

### ISSUE THREAD

**Issue by stef-barburou** [stef-barburou/countdowntogame#4]
to be able to use this, we need to publish it to github pages

**Comment by stef-barburou** [comment]
yeah but the problem is, stef's github page is being used right now, @smoothpickle  do you know if it's possible to publish from this repository into your smoothpickle.github.io ? (github pages)

**Comment by Gargelspouzel** [comment]
can I do it? I want to help!
